src/include/cursor.i

```diff
@@ -174,12 +174,10 @@ static inline void
 __cursor_leave(WT_SESSION_IMPL *session)
 {
     /*
-     * Decrement the count of active cursors in the session. When that goes to zero, there are no
-     * active cursors, and we can release any snapshot we're holding for read committed isolation.
```

Comment:
SIngle line comments generally use the form:

```
/* Decrement the count of active cursors in the session. */
```

Similarly above.

Reply:
Fixed